orchestrator/orchestrator.go

```diff
@@ -230,6 +233,20 @@ func (s *Orchestrator) resolveInput(wfHash hash.Hash, exec *execution.Execution,
 	return exec.Outputs.Fields[outputKey], nil
 }
 
+func (s *Orchestrator) resolveResult(wfHash hash.Hash, exec *execution.Execution, instanceHash hash.Hash, outputKey string) (*types.Value, error) {
+	if !wfHash.Equal(exec.ProcessHash) {
```

Comment:
i'm not sure but i think `exec.ProcessHash` will never contains the same `wfHash` as it is created manually or by another process.

Reply:
Can you add test about resolve result. It will help me a lot with implementing that logic and future refactor.